packages/components/src/components/select/select.lite.tsx

```diff
@@ -233,7 +233,9 @@ export default function DBSelect(props: DBSelectProps) {
 				}
 				aria-describedby={props.ariaDescribedBy ?? state._descByIds}>
 				{/* Empty option for floating label */}
-				<option hidden></option>
+				<Show when={props.variant === 'floating' || props.placeholder}>
+					<option class="placeholder" value=""></option>
```

Comment:
```suggestion
					<option class="placeholder" value="" disabled></option>
```
If it is not wanted to select the empty value, we can set 'disabled', so the value can't be selected and submitted.